### ISSUE TITLE
the loss didn't decrease.

### ISSUE THREAD

**Issue by Disyxt** [ml-struct-bio/drgnai#28]
hello,I have trained a few epochs, but the loss didn't decrease.
The loss of first epoch is about 0.85, and the loss of 37th epoch is about 0.85 too.
What's wrong with my training?

<img width="623" height="256" alt="Image" src="https://github.com/user-attachments/assets/da3003b4-48e8-4b7b-90d0-2259d3b2d3b2" />

**Comment by michal-g** [comment]
Hi, a similar question was recently posted in the issues forum for our sibling software package [cryoDRGN](https://github.com/ml-struct-bio/cryodrgn), so I'll repost my answer from there here as it applies equally as well:

> ...changes in loss between training epochs do not necessarily correlate with updates to reconstruction quality nor model convergence due largely to the high-noise regime under which cryo-EM reconstruction models operate. We recommend instead comparing the outputs of the models themselves, as loss can indeed stop decreasing by significant amounts well before the model has stopped making significant improvements to these outputs!
> 
> Have you looked at the output volumes — and if doing heterogeneous reconstruction, the latent spaces — produced by tools such as `analyze` on various epochs from your models? This should give a much better idea of how your models are performing than only looking at the training loss! 
>

 _Originally posted by @michal-g in [#459](https://github.com/ml-struct-bio/cryodrgn/issues/459#issuecomment-3445135039)_

I would add that in this case, if poses are available, it would be a good idea to run `cryodrgn backproject_voxel` on your inputs as a sanity check to make sure they were properly pre-processed!

Let us know if this helps!
-Mike
